src/common/file_buffer.cpp

```diff
@@ -100,6 +100,10 @@ void FileBuffer::Resize(BlockManager &block_manager) {
 	ResizeInternal(block_manager.GetBlockSize(), block_manager.GetBlockHeaderSize());
 }
 
+void FileBuffer::Resize(const BlockManager &block_manager) {
+	ResizeInternal(block_manager.GetBlockSize(), block_manager.GetBlockHeaderSize());
```

Comment:
nit: I think we can just remove the method above with the non-const BlockManager now